### DISCUSSION TITLE
Module not found: Can't resolve '***\app\favicon.ico?__next_metadata__'

### DISCUSSION THREAD

**Discussion by Sir-Erwin** [vercel/next.js#70297] [Help]
### Link to the code that reproduces this issue

https://github.com/Sir-Erwin/project-xenia.git

### To Reproduce

1. npm run dev

(after compiling it starts to throw the following error)

### Current vs. Expected behavior

I expected to see the code successfully seeing that I had made no modifications from what I got from npx create-next-app@latest.

However, it threw an error saying " Module not found: Can't resolve '***\project-xenia\app\favicon.ico?__next_metadata__' "

### Provide environment information

```bash
Operating System:
  Platform: win32
  Arch: x64
  Version: Windows 11 Home
  Available memory (MB): 12085
  Available CPU cores: 8
Binaries:
  Node: 20.11.1
  npm: N/A
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 14.2.11 // Latest available version is detected (14.2.11).
  eslint-config-next: N/A
  react: 18.3.1
  react-dom: 18.3.1
  typescript: N/A
Next.js Config:
  output: N/A
```

### Which area(s) are affected? (Select all that apply)

create-next-app, Image (next/image), Metadata

### Which stage(s) are affected? (Select all that apply)

next dev (local)

### Additional context

I even tested it out with the previous 14.x.x versions, and even a 13.5.x version but it's throwing the same error.

I tried removing favicon.ico, but it just throws more errors. And sometimes it doesn't even start compiling, or freezes when compiling. I need to kill the processes eventually.

When I tried reproducing this in sandbox tho, there didn't seem to be a problem.

My guess, it has something to do with Windows, like its paths and stuff...

**Comment by Sir-Erwin** [comment]
When I delete favicon.ico, I get the following error

Module not found: Can't resolve 'next/font/local/target.css?{"path":"app\\layout.tsx","import":"","arguments":[{"src":"./fonts/GeistVF.woff","variable":"--font-geist-sans","weight":"100 900"}],"variableName":"geistSans"}

**Comment by samcx** [comment]
@Sir-Erwin Can you try the latest canary to see if you're experiencing with that as well?

Will have to install the release candidate of `react` and `react-dom` as well.

**Comment by Sir-Erwin** [comment]
Same response

![image](https://github.com/user-attachments/assets/bd58c887-1daf-4ef9-a102-8c2370ea6a3a)

**Comment by samcx** [comment]
@Sir-Erwin Just to confirm, this is with the <picture data-single-emoji=":repro:" title=":repro:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/repro/5cea53b5532825b4.png" alt=":repro:" align="absmiddle"></picture> as well? I'm surprised it's taking 15.5s to compile <picture data-single-emoji=":wow:" title=":wow:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/wow/8524bc5963b93848.gif" alt=":wow:" align="absmiddle"></picture>. 

Also, which package manager are you using (since `next info` is showing N/A is showing empty for all of them)?

**Comment by Sir-Erwin** [comment]
Wait, what's repro...

I don't know, I just used "npx create-next-app@latest"

I had downloaded node and react about 6 months ago, but they seem to be up to date...

**Comment by samcx** [comment]
@Sir-Erwin Can you share with us what was above your screenshot (your entire terminal process from starting the `next dev` Development server)?

**Comment by Sir-Erwin** [comment]
![image](https://github.com/user-attachments/assets/2e1c5027-fbd6-4da2-ad00-b6fe836eee2e)

![image](https://github.com/user-attachments/assets/773e97cf-cc57-4a58-a9de-5ef687b3740e)

![image](https://github.com/user-attachments/assets/1965b489-faf7-4560-8f73-952e1c8ce98a)

**Comment by samcx** [comment]
@Sir-Erwin Can you try installing a package manager (e.g., `npm`) locally to see if that makes a difference?

**Reply by Sir-Erwin** [reply]
Doesn't npm come with Node, how do I install it locally?

**Reply by samcx** [reply]
I believe so. Maybe try `npm run dev` instead of `npx next dev`?

**Reply by Sir-Erwin** [reply]
It's the same result

**Comment by wordopera** [comment]
The post caught my eye.
My next.js 14.2.11 was showing favicon 404 messages in the terminal. No 404 errors on the web page. They were showing up with different preceding file paths. e.g. /auth/form and app/components. That seemed weird so didn't want to ignore.

Note I had favicon in the intended folder at /public.

I had favicon in /public. chatgpt sent me to check middleware tsconfig but was happy with those. I searched for favicon and found it in long forgotten metadata.ts. Found this " icons: {
icon: './favicon',

},"

I removed the dot and removed the comma. So it became "icon: '/favicon' },"
Never saw the 404 message again.

**Reply by Sir-Erwin** [reply]
I looked for a similar issue, but couldn't find it
